src/components/atoms/descriptors/descriptors.tsx

```diff
@@ -0,0 +1,23 @@
+import './descriptors.scss';
+
+export const Descriptors = ({ doi }: { doi:string }): JSX.Element => (
+  <div className="descriptors">
+    <ul className="descriptors__identifiers">
+      <li className="descriptors__identifier">
+        <a href={`https://doi.org/${doi}`}>https://doi.org/{doi}</a>
+      </li>
+    </ul>
+    <ul className="descriptors__icons">
+      <li>
+        <a href="https://en.wikipedia.org/wiki/Open_access" className="descriptors__icon descriptors__icon--oa">
+          <span className="visuallyhidden">Open access</span>
+        </a>
+      </li>
+      <li>
+        <a href="https://creativecommons.org/licenses/by/4.0/" className="descriptors__icon descriptors__icon--cc">
+          <span className="visuallyhidden">Copyright information</span>
```

Comment:
Will all reviewed preprints by published under the same license? Of course, we can determine that this is the case for the initial examples.

Reply:
This would be a question for the production team i think, we modelled this on the version we had before. It may be an enhancement that we worry about after Oct (or at least when we know we need it).